### ISSUE TITLE
Unable to save filters

### ISSUE THREAD

**Issue by shawniverson** [mailwatch/1.2.0#638]
```
Invalid query: 1136: Column count doesn't match value count at row 1
Whole query:

REPLACE INTO `saved_filters` VALUES ('test','date','=','2017-03-12','postmaster')
```

It appears that the missing id field is expected in the replace query.

**Comment by Skywalker-11** [comment]
Where exactly does that error occure?

**Comment by asuweb** [comment]
I've come across this on other queries in MW where the db schema has been changed, however the queries themselves have not been checked. 

We should be specifying the fields in the query to prevent this from happening.

**Comment by shawniverson** [comment]
It is the main filter form under Search and Reports.  Using the latest develop, create a filter, and try to save it.

**Comment by asuweb** [comment]
Fixed in #639

**Comment by asuweb** [comment]
This can be closed
